doc/security/specs/secure_boot/index.md

```diff
@@ -86,6 +86,17 @@ The roles are:
 If the key indicated in the manifest has a role that doesn't match the lifecycle state of the device, the boot fails.
```

Comment:
Should we split this document change out to a separate PR?

- We need to add a section (or a separate doc) that discusses what a "full complement of keys" means.
- We need to discuss what the full complement of keys will be for Earlgrey.

Reply:
Yes I will split this PR into two.

Reply:
I have split the documentation into a separate PR #17449